core/variant/method_ptrcall.h

```diff
@@ -179,6 +179,38 @@ struct PtrToArg<const T *> {
 	}
 };
 
+// This is for RequiredPtr.
+
+template <class T>
+struct PtrToArg<RequiredPtr<T>> {
+	typedef T *EncodeT;
+
+	_FORCE_INLINE_ static T *convert(const void *p_ptr) {
+		if (p_ptr == nullptr) {
+			// Should we show an error?
+			return RequiredPtr<T>(nullptr);
+		}
+		return RequiredPtr<T>(const_cast<T *>(*reinterpret_cast<T *const *>(p_ptr)));
```

Comment:
Is the `const_cast` needed here?

The `reinterpret_cast` returns a type `T *const *`, which is then dereferenced into `T* const`.
That is a top-level CV qualifier and can as such be directly cast into `T*`.

It's _not_ `T const*` (== `const T*`), which _would_ need a `const_cast`.

Reply:
Ah, I think you're right - I've removed the `const_cast` in my latest push